notebooks/tuning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Perhaps it is just me - but I find it a bit confusing that this is listed as a separate point from <code>ForecastingGridSearchCV</code> since <code>MultiplexForecaster</code> is ultimately tuned using <code>ForecastingGridSearchCV</code> What if instead we framed it as - we can 1) tune parameters within the various steps of the pipeline or 2) try out different models all together? Would allow you to also desicribe <code>MultiplexTransformer</code> as part of 2) as well which I think could make sense.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>The difference of <code>MultiplexForecaster</code> approach is that you cant tune the other parameters anymore. So thats why it might be good to have it separate? Its a different use case scenario - when you already have tuned forecasters and just want to compare them with a cross-validation. </p>